servers/quarkus-server/pom.xml

```diff
@@ -165,6 +165,14 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-container-image-jib</artifactId>
     </dependency>
+    <dependency>
```

Comment:
what does this dependency provide? Do we really want to ship all servers w/ this?

Reply:
the purpose of this is to **only** generate kubernetes manifest files during the build

Reply:
do they get packaged into the jar file or docker image or just left in `target`?

Reply:
they will just stay in `target` and are not making it into the jar